docs/zh/latest/FAQ.md

```diff
@@ -594,6 +594,6 @@ apisix:
 
 ## 如果在使用 APISIX 过程中遇到问题，我可以在哪里寻求更多帮助？
 
-- [Apache APISIX Slack Channel](/docs/general/community#joining-the-slack-channel)：加入后请选择 channel-apisix 频道，即可通过此频道进行 APISIX 相关问题的提问。
-- [邮件列表](/docs/general/community#joining-the-mailing-list)：任何问题或对项目提议都可以通过社区邮件进行讨论。
+- [Apache APISIX Slack Channel](/docs/general/join/#加入-slack-频道)：加入后请选择 channel-apisix 频道，即可通过此频道进行 APISIX 相关问题的提问。
```

Comment:
Because this page won't be too long, so I think just use `/join` is good enough 😊 but LGTM anyway.